examples/post_training_quantization/torch/fx/resnet18/main.py

```diff
@@ -0,0 +1,225 @@
+    for image_filename, image_label in val_annotation_data:
+        from_image_filepath = val_images_dir / image_filename
+        to_image_dir = val_data_dir / image_label
+        if not to_image_dir.exists():
+            to_image_dir.mkdir()
+        to_image_filepath = to_image_dir / image_filename
+        from_image_filepath.rename(to_image_filepath)
+    val_annotations_file.unlink()
+    val_images_dir.rmdir()
+
+
+def main():
+    device = torch.device("cpu")
+    print(f"Using {device} device")
+
+    ###############################################################################
+    # Step 1: Prepare model and dataset
+    print(os.linesep + "[Step 1] Prepare model and dataset")
+
+    model = get_resnet18_model(device)
+    model, acc1_fp32 = load_checkpoint(model)
+
+    print(f"Accuracy@1 of original FP32 model: {acc1_fp32}")
+
+    val_loader, calibration_dataset = create_data_loaders()
+
+    def transform_fn(data_item):
+        return data_item[0].to(device)
+
+    quantization_dataset = nncf.Dataset(calibration_dataset, transform_fn)
+
+    ###############################################################################
+    # Step 2: Quantize model
+    print(os.linesep + "[Step 2] Quantize model")
+
+    input_shape = (1, 3, IMAGE_SIZE, IMAGE_SIZE)
+    example_input = torch.ones(*input_shape).to(device)
+
+    fx_model = torch.export.export(model.eval(), args=(example_input,)).module()
+    quantized_fx_model = nncf.quantize(fx_model, quantization_dataset)
+
+    quantized_fx_model = torch.compile(quantized_fx_model, backend="openvino")
+
+    acc1_int8 = validate(val_loader, quantized_fx_model, device)
+    print(f"Accuracy@1 of INT8 model: {acc1_int8:.3f}")
+    print(f"Accuracy diff FP32 - INT8: {acc1_fp32 - acc1_int8:.3f}")
+
+    ###############################################################################
+    # Step 3: Run benchmarks
+    print(os.linesep + "[Step 3] Run benchmarks")
+    print("Run benchmark for FP32 model compiled with default backend ...")
+    compiled_model = torch.compile(model)
+    fp32_latency = measure_latency(compiled_model, example_inputs=example_input)
+    print(f"{fp32_latency:.3f} ms")
+
+    print("Run benchmark for FP32 model compiled with openvino backend ...")
+    compiled_model = torch.compile(model, backend="openvino")
+    fp32_ov_latency = measure_latency(compiled_model, example_inputs=example_input)
+    print(f"{fp32_ov_latency:.3f} ms")
+    print(f"Speed up relative to FP32 model with default backend: x{fp32_latency / fp32_ov_latency:.3f}")
+
+    print("Run benchmark for INT8 model compiled with openvino backend ...")
+    int8_latency = measure_latency(quantized_fx_model, example_inputs=example_input)
+    print(f"{int8_latency:.3f} ms")
+    print(f"Speed up relative to FP32 model with default backend: x{fp32_latency / int8_latency:.3f}")
+    print(f"Speed up relative to FP32 model with openvino backend: x{fp32_ov_latency / int8_latency:.3f}")
```

Comment:
```suggestion
    print("Benchmark FP32 model compiled with default backend ...")
    compiled_model = torch.compile(model)
    fp32_latency = measure_latency(compiled_model, example_inputs=example_input)
    print(f"Latency:{fp32_latency:.3f} ms")

    print("Run benchmark for FP32 model compiled with openvino backend ...")
    compiled_model = torch.compile(model, backend="openvino")
    fp32_ov_latency = measure_latency(compiled_model, example_inputs=example_input)
    print(f"Latency: {fp32_ov_latency:.3f} ms")

    print("Run benchmark for INT8 model compiled with openvino backend ...")
    int8_latency = measure_latency(quantized_fx_model, example_inputs=example_input)
    print(f"Latency: {int8_latency:.3f} ms")
    
    print("[Step 4] torch.compile(..., backend="openvino") speed ups:"
    print(f"FP32 model: x{fp32_latency / fp32_ov_latency:.3f}")
    print(f"INT8 model: x{fp32_ov_latency / int8_latency:.3f}")
```

Reply:
I fixed minor mistakes and aligned all prints as it done with `Benchmark FP32 model compiled with default backend ...`
Please check